src/components/MergeInfographic.tsx

```diff
@@ -82,22 +31,46 @@ const SvgText: React.FC<SvgProps> = ({ getString }) => {
       xmlnsXlink="http://www.w3.org/1999/xlink"
       aria-hidden="true"
     >
-      <Text x="2%" y="35%" fontSize={lg}>
+      <text x="2%" y="35%" fontSize={lg} textAnchor="start" fill="currentColor">
```

Comment:
With all of the `text` elements, you could do the same as the `Background` svg in using the Chakra factory, so you can keep the code DRY with the `textAnchor` and `fill` props.

```tsx
const Text = chakra("text", {
  baseStyle: {
    textAnchor: "start",
    fill: "currentColor",
  },
})
```